apps/speed.c

```diff
@@ -3508,7 +3508,11 @@ static int do_multi(int multi, int size_num)
         char buf[1024];
         char *p;
 
-        f = fdopen(fds[n], "r");
+        if ((f = fdopen(fds[n], "r")) == NULL){
+            BIO_printf(bio_err, "fdopen failure with 0x%x\n",
+                           errno);
+            return 1;
+        }
```

Comment:
The code looks good, but there's formatting issues. I will note that other error messages don't include `errno`.